### PR TITLE
Vijayasree/refactor: academy and template page with new show component

### PR DESCRIPTION
**Pull request by vijayasree-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by NikitaK-deriv — DISMISSED** [review]

File: src/pages/academy/blog/posts/preview/index.tsx [hunk 1]
```diff
@@ -160,7 +160,7 @@ const BlogPreview = () => {
                                     <InfoText size="14px" mt="16px">
                                         {getMinRead(post_data?.blog_post)}
                                     </InfoText>
-                                    <Show.Mobile min_width="laptop">
+                                    <Mobile breakpoint={'laptop'}>
```
Comment (sean-binary) [inline src/pages/academy/blog/posts/preview/index.tsx:163]:
```suggestion
                                    <Mobile breakpoint="laptop">
```

Comment (sean-binary) [reply src/pages/academy/blog/posts/preview/index.tsx:163]: Same for the rest 🙏

Comment (vijayasree-deriv) [reply src/pages/academy/blog/posts/preview/index.tsx]: updated @sean-binary

File: src/components/containers/visibility.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ import { useWebsiteStatus } from 'components/hooks/use-website-status'
 
 type ResponsiveContainerProps = {
     children: ReactElement
-    breakpoint?: number
```
Comment (sean-binary) [inline src/components/containers/visibility.tsx:12]: You'll need to resolve the conflicts that will come from #2771 later.

**Review by sean-binary — APPROVED** [review]

File: src/components/containers/visibility.tsx (hunk 2 above)
Comment (sean-binary) [reply src/components/containers/visibility.tsx:12]: As discussed, this will be done later in #2771.

**Review by aswathy-deriv — APPROVED** [review]
LGTM

**Pull request merged**